_posts/elements/2015-04-09-buttons.md

```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
 info: Buttons should be used for performing actions within the edX environment. While we supply a button that looks like a link, it should only really be used for very tertiary actions. edX offers three button sizes, each with normal, hover, active/pressed, and disabled states. We also offer a range of other buttons to suit your needs.
 ---
 
+<h3 class="hd-6 example-set-hd">Default</h3>
 <section class="example-set">
-    <h3 class="hd-6">Default</h3>
     <button type="button" class="btn btn-default btn-base">Default Base</button>
     <button type="button" class="btn btn-default btn-large">Default Large</button>
     <button type="button" class="btn btn-default btn-small">Default Small</button>
     <button type="button" class="btn btn-default btn-base" disabled>Default Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Default (Alt)</h3>
 <section class="example-set">
-    <h3 class="hd-6">Default (Alt)</h3>
     <button type="button" class="btn btn-default-alt btn-base">Default Base</button>
     <button type="button" class="btn btn-default-alt btn-large">Default Large</button>
     <button type="button" class="btn btn-default-alt btn-small">Default Small</button>
     <button type="button" class="btn btn-default-alt btn-base" disabled>Default Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Primary</h3>
 <section class="example-set">
-    <h3 class="hd-6">Primary</h3>
     <button type="button" class="btn btn-primary btn-base">Primary Base</button>
     <button type="button" class="btn btn-primary btn-large">Primary Large</button>
     <button type="button" class="btn btn-primary btn-small">Primary Small</button>
     <button type="button" class="btn btn-primary btn-base" disabled>Primary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Secondary</h3>
 <section class="example-set">
-    <h3 class="hd-6">Secondary</h3>
```

Comment:
Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.